### DISCUSSION TITLE
Unable to reactivate student account

### DISCUSSION THREAD

**Discussion by ssolooo** [community/community#143528] [GitHub Education]
### Hi everyone,

The Student Pack has expired and i'm unable to reverify my details to reactivate it.

When i try to hit the "Continue" button is not working, even after clearing your cache. A pop-up appeared to access my location, i allowed it and after that the button is disabled and nothing happend next, how can i solve this?

**Comment by raymon-roos** [comment]
I'm experiencing the same, I tried it on both a firefox-based as chromium-based browser.
